easybuild/easyconfigs/g/GenomeWorks/GenomeWorks-2021.02.2-fosscuda-2020b.eb

```diff
@@ -0,0 +1,135 @@
+        'download_filename': 'd0bafdb9fdde2ee99932d42f1deec74d53e19514.tar.gz',
+    },
+    {
+        'source_urls': ['https://github.com/rvaser/spoa/archive'],
+        'filename': 'spoa-20190416.tar.gz',
+        'download_filename': '0b9e56da3241d6a887d64adcfd1ebfae29810c04.tar.gz',
+    },
+]
+checksums = [
+    'f036f066b3ae22e603786f79d374cde28942c22064a169f123fb085fd6b4ccaf',  # v2021.02.2.tar.gz
+    'f68aec93154d010324c05bcd8c5cc53468b87af88d87acb5ddcfaa1bba044837',  # benchmark-20190513.tar.gz
+    '7831843b8ab5f9d9dfbcea3b9ad2632a229c6d1aa9a5a335b8d8b813707dd9ec',  # cub-20201123.tar.gz
+    '6fcf88c123bdee5cbef5f1d3b2b34fa6f20cb3168bf05a2adbe20428cae3fe0e',  # googletest-20190424.tar.gz
+    '28455110611f6330b5d6b49ff5c3c1c824a7727944c66c8c6f0e9ce1209740ea',  # htslib-20200806.tar.gz
+    'ae64499023d4705f42e79a966f175c31a46a794a1dd6fe9c03dd1827efe4a953',  # kseqpp-20181009.tar.gz
+    '0427eaf2c5deabdeeb7f1a894fd362823e32450d6c2c79fe20e64a8898968ef9',  # libcudacxx-20201003.tar.gz
+    '9608c0d31dc158aaab8916b8977962f88e42d2341f979d517daac03db525ea70',  # spoa-20190416.tar.gz
+]
+
+builddependencies = [('CMake', '3.18.4')]
+dependencies = [
+    ('Python', '3.8.6'),
+    ('Autoconf', '2.69'),
+    ('matplotlib', '3.3.3'),
+    ('SciPy-bundle', '2020.11'),
+    ('networkx', '2.5'),
+    ('tqdm', '4.56.2'),
+]
+local_vendor_dir = '%(builddir)s/GenomeWorks-*/3rdparty/'
+# benchmark
+preconfigopts = "mv %%(builddir)s/benchmark-*/* %s/benchmark/ && " % local_vendor_dir
+# cub
+preconfigopts += "mv %%(builddir)s/cub-*/* %s/cub/ && " % local_vendor_dir
+# googletest
+preconfigopts += "mv %%(builddir)s/googletest-*/* %s/googletest/ && " % local_vendor_dir
+# htslib
+preconfigopts += "mv %%(builddir)s/htslib-*/* %s/htslib/ && " % local_vendor_dir
+# kseqpp
+preconfigopts += "mv %%(builddir)s/kseqpp-*/* %s/kseqpp/ && " % local_vendor_dir
+# libcudacxx
+preconfigopts += "mv %%(builddir)s/libcudacxx-*/* %s/libcudacxx/ && " % local_vendor_dir
+# spoa
+preconfigopts += "mv %%(builddir)s/spoa-*/* %s/spoa/ && " % local_vendor_dir
+
+# add missing include to a file
+preconfigopts += "sed -i'' 's/#include <exception>/#include <exception>\\n#include <stdexcept>/g'"
+preconfigopts += " %(builddir)s/GenomeWorks-*/3rdparty/spoa/src/alignment_engine.cpp && """
+
+configopts = "-Dgw_cuda_gen_all_arch=OFF"
+
+skipsteps = ['test']
+
+exts_defaultclass = 'PythonPackage'
+
+exts_default_options = {
+    'source_urls': ['https://github.com/clara-parabricks/GenomeWorks/archive/'],
+    'download_dep_fail': True,
+    'use_pip': True,
+}
+
+exts_list = [
+    ('genomeworks', version, {
+        'patches': ['GenomeWorks-2021-02-2_dep_versions_n_file_extension.patch'],
+        'postinstallcmds': ["""
+                            sed -i'' 's/==/>=/g' %(builddir)s/GenomeWorks-%(version)s/pygenomeworks/requirements.txt
+                            """],
+        'preinstallopts': """export GW_INSTALL_DIR=%(installdir)s &&
```

Comment:
define `local_genomeworks_preinstallopts` above `exts_list` to avoid triple quoting here:

```python
local_genomeworks_preinstallopts = "export GW_INSTALL_DIR=%(installdir)s && "
local_genomeworks_preinstallopts += "export GW_VERSION=%(version)s && "
local_genomeworks_preinstallopts += "export GW_ROOT_DIR=%(builddir)s/GenomeWorks-%(version)s"

exts_list = [
...
        'preinstallopts': local_genomeworks_preinstallopts,